x-pack/test/plugin_api_integration/test_suites/task_manager/skip.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SerializedConcreteTaskInstance, TaskStatus } from '@kbn/task-manager-plugin/server/task';
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const log = getService('log');
+  const retry = getService('retry');
+
+  describe('Skip invalid tasks', () => {
```

Comment:
Seems like we need a test where the invalid indirect params suddenly become valid - kinda the opposite of the alerting test that was deleted where the params became invalid.  To test that the task WILL get another chance to succeed, and that it will.  Fine to add in a separate PR before FF.